config_spec.toml

```diff
@@ -129,3 +129,9 @@ name = "cashaccount_activation_height"
 type = "usize"
 doc = "The activation blockheight for cashaccount. Set to 0 to disable cashaccount indexing"
 default = "563720"
+
+[[param]]
+name = "rpc_buffer_size"
+type = "usize"
+doc = "Size of the message queue for each peer. If set to small, subscription notifications may drop"
```

Comment:
I'd add an example for a "too small" value

Reply:
I'm not sure what too small value is, other than that the current value we use is too small for sure.